### DISCUSSION TITLE
求帮助，所有订阅拉取失败咋回事？设置了代理也不行。。以前还可以的。

### DISCUSSION THREAD

**Discussion by jinnian36** [whyour/qinglong#2184] [Q&A]
开始拉取仓库 mrabit_aliyundriveDailyCheck 到 /ql/data/repo/mrabit_aliyundriveDailyCheck

Cloning into '/ql/data/repo/mrabit_aliyundriveDailyCheck'...

fatal: Out of memory, malloc failed (tried to allocate 524288000 bytes)

fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

拉取 mrabit_aliyundriveDailyCheck 失败，请检查网络...

**Comment by Martinxux** [comment]
同样的问题，有解决吗

**Reply by jinnian36** [reply]
解决了，但是我忘了，我只记得CSDN上面有解决办法

**Reply by Martinxux** [reply]
`git config --global --unset http.postBuffer`

试了下这个可以，但是原因不清楚。
按理说`http.postBuffer`应该没有手动设置过，不知道为啥被设置了。取消`http.postBuffer`就行了

**Comment by Chunhui-Zou** [comment]
同样的问题，请问你解决了吗？
git config --global --unset http.postBuffer 没有效果

**Reply by Martinxux** [reply]
你是在青龙容器终端里面输的吗
